### PR TITLE
readme: small fixes

### PR DESCRIPTION
**Pull request by hinto-janaiyo**
very small changes to the readme

- empty spaces after sentences removed
- dependency install command blocks made copyable like the other ones below it
- backtick typo fix

sorry if reviewing this is a waste of brain power

### CODE REVIEW

**Review by selsta — APPROVED** [review]

**Review by mj-xmr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Monero
 
-Copyright (c) 2014-2022 The Monero Project.   
+Copyright (c) 2014-2022 The Monero Project.
```
Comment (tobtoht) [inline README.md:3]: The double spaces are needed here to produce a newline. Keep them.

Comment (hinto-janaiyo) [reply README.md]: Ah, I was wondering what they were for in the first place, thanks.

**Review by selsta — COMMENTED** [review]
Do you know how to squash your commits to a single commit?

**Review by mj-xmr — APPROVED** [review]

**Comment by selsta** [comment]
@hinto-janaiyo sorry, one more thing :D could you name the commit title as `readme: small fixes` instead of the branch name? then it's good to merge

**Review by selsta — APPROVED** [review]

**Pull request merged**